File: projects/m1/007-day-old-bread/solutions/GabrieleGabellone/day_old_bread.py
#Day-old-bread

loaves = int(input("Enter the number of the loaves: "))
regular_price = 3.49 * loaves
discount = regular_price/100*60
total_price = regular_price - discount
regular_price = "{:.2f}".format (regular_price)
discount = "{:.2f}".format (discount)
total_price = "{:.2f}".format (total_price)
print ("The regular price of bread is: €", regular_price)
print ("You have a discount of €", discount, "because it is one day old")
print ("So, the total price is €", total_price)

File: projects/m1/007-day-old-bread/solutions/vittorioMenicanti/dayOldBread.js


//Day Old Bread

const breadQuantities = parseFloat(prompt('Enter quantities of bread:'));

const breadPrice = 3.49;

const oldBread = 60;

let totalPrice = breadQuantities * breadPrice;

let discount = (oldBread/100)*breadPrice;

let discountedPrice = totalPrice - discount;

console.log(`Bread Quantities Purchased: ${breadQuantities} Breads`);

console.log(`Bread Price is: ${breadPrice.toFixed(2)}`);

console.log(`Total price is: ${totalPrice.toFixed(2)}`);

console.log(`Discount applied is: ${60}%`);

console.log(`Total dicounted price is: ${discountedPrice.toFixed(2)}`);

console.log(`Thanks for buying at our place,See you!`);

